api/src/test/java/org/openmrs/api/VisitServiceTest.java

```diff
@@ -437,6 +456,20 @@ public void getActiveVisitsByPatient_shouldReturnAllUnvoidedVisitsForTheSpecifie
 		assertEquals(8, visitService.getVisitsByPatient(new Patient(2), true, false).size());
 	}
 	
+	@Test
+	public void getActiveVisitsByPatient_shouldReturnEmptyListGivenNull() {
+		assertThat(visitService.getVisitsByPatient(null, false, true), is(empty()));
+	}
+	
+	@Test
+	public void getActiveVisitsByPatient_shouldReturnEmptyVisitsForNullPatientId() {
+		Patient p = new Patient();
+		p.setPatientId(null);
+		
+		assertThat(visitService.getVisitsByPatient(p, false, true), is(empty()));
+		
```

Comment:
remove newline.

be consistent in how you write your code. either always add a newline at the end of a test or not. this makes it easier to read. I prefer to add a newline at the beginning of the test because I think it increases readability since it separates the code from the long method name. I dont think a newline at the end before the closing `}` improves readability.